test/incident-72/verify-heal-in-memory.td

```diff
@@ -43,6 +43,6 @@ $ set-from-sql var=source-id
 SELECT id FROM mz_objects WHERE name = 'textavro';
 
 $ set-from-sql var=source-shard-id
-SELECT shard_id FROM mz_internal.mz_storage_shards WHERE object_id = '${source-id}';
+SELECT shard_id FROM mz_unsafe.mz_storage_shards WHERE object_id = '${source-id}';
```

Comment:
It seems like this shouldn't have changed?

Reply:
Yep, fixed.